cantusdb/turtle/csv2rdf_by_RDFLib.py

```diff
@@ -3,64 +3,60 @@
 import csv
 import validators
 import sys
+import json
 
 # helper to get the predicate relation dictionary
 def get_ontology_dict () :
-    
-    # a specific config file for the predicates
-    # TODO: maybe change this part to make a more customized way of getting the predicates configurations
-    ontology_f = open("ontology_dict.csv", 'r')
-    ontology_reader = csv.DictReader(ontology_f)
-    ontology_dict = {}
-    
-    # converting the reader dict to the predicate dictionary
-    for row in ontology_reader:
-        ontology_dict[row[list(row.keys())[0]]] = row[list(row.keys())[1]]
-                
-    return ontology_dict
+    with open("relations_mapping.json", 'r') as mapper:
+        # a specific config file for the predicates
+        # TODO: maybe change this part to make a more customized way of getting the predicates configurations
+        data = json.load(mapper)
+    return data
     
 # a main operation function
 def convert_csv_to_turtle():
-    # TODO: maybe change this one to accomadate for multiple files
     if len(sys.argv) != 2:
-        raise ValueError("Invalid number of input filename") 
-    
-    g = Graph()
-
-    csv_file = open(sys.argv[1], 'r', encoding='utf-8')
-    
-    csv_reader = csv.DictReader(csv_file)
-    ontology_dict = get_ontology_dict()
+            raise ValueError("Invalid number of input filename") 
```

Comment:
What is the vision for how this utility is to be used? As a command line tool? If so, it would be good for the arguments to have help strings, checks on type, etc. (for example, take a look at python's argparse module). Is it as a python module? If so, then this function should take some arguments.

Reply:
In the future, it might be used as a command in some executable. I will add help strings and type checks.

Reply:
I think one good check for a PR is whether a reviewer could test it... and in this case I'd need some more information either in the code or in documentation about what and how to pass in what needs to be passed.